### DISCUSSION TITLE
Alpine Expression Error when using wire:navigate

### DISCUSSION THREAD

**Discussion by francoism90** [livewire/livewire#6764] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.2

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/francoism90/hub-api

### Steps To Reproduce

1. Add https://livewire.laravel.com/docs/navigate to `a` element
3. On page use `<div x-init x-data="{ open: false }" />`
4. Navigate to page using `a` tag, and use browser's navigation to forward/go back
5. See console output error

### Additional information

```
Alpine Expression Error: 'open' called on an object that does not implement interface Window.

Expression: "open"
```
It seems to happen because it doesn't re-render the page (correctly).

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
We're in the process of reviewing all of the navigate issues. I've just tested this and the issue seems to be fixed, so going to close this. If you're still having issues, please open a new bug discussion thanks!
